definitions/output/reports/cwv_tech_technologies.js

```diff
@@ -6,19 +6,68 @@ publish('cwv_tech_technologies', {
   tags: ['crux_ready']
 }).query(ctx => `
 /* {"dataform_trigger": "report_cwv_tech_complete", "name": "technologies", "type": "dict"} */
+WITH pages AS (
+  SELECT DISTINCT
+    client,
+    root_page AS origin,
+    tech.technology
+  FROM ${ctx.ref('crawl', 'pages')},
+    UNNEST(technologies) AS tech
+  WHERE
+    date = '${pastMonth}'
+    ${constants.devRankFilter}
+), tech_origins AS (
```

Comment:
Nit but can we separate these out to be a bit more readable?

```suggestion
),

tech_origins AS (
```